apps/db/priv/repo/migrations/20220118101217_allow_netex_conversion.exs

```diff
@@ -0,0 +1,13 @@
+defmodule DB.Repo.Migrations.AllowNetexConversion do
+  use Ecto.Migration
+
+  def up do
+    drop constraint("data_conversion", :allowed_to_formats)
+    create constraint("data_conversion", :allowed_to_formats, check: "convert_to IN ('GeoJSON', 'NeTEx')")
+  end
```

Comment:
Pas moyen de lister des combinaisons de possibilités ?

Avec de la concaténation :trollface: `(convert_from || '-' || convert_to) in ('GTFS-GeoJSON', 'GTFS-NeTEx')` 💩

Reply:
Pour l'instant on ne convertit que du GTFS, donc ça ne pose pas de problème.
J'ai l'impression que si les conversions se multiplient de X formats vers Y formats, le plus simple sera de faire les vérifications côté Elixir avec un `Changeset`, et là on pourra vérifier facilement ce genre de choses.
La contrainte en base, je la sens un peu overkill rétrospectivement.

Reply:
Oui au pire un petit validate via Ecto ça pourrait suffire je pense...

Reply:
je le ferai la prochaine fois qu'on ajoutera une conversion...s'il y en a une :)